src/handlers/content-type-handlers.ts

```diff
@@ -2,21 +2,30 @@
 import { getContentfulClient } from "../config/client.js"
 import { ContentTypeProps, CreateContentTypeProps } from "contentful-management"
 import { toCamelCase } from "../utils/to-camel-case.js"
+import {summarizeData} from "../utils/summarizer.js";
 
 export const contentTypeHandlers = {
-  listContentTypes: async (args: { spaceId: string; environmentId: string }) => {
+  listContentTypes: async (args: { spaceId: string; environmentId: string, limit?: number, skip?: number }) => {
     const spaceId = process.env.SPACE_ID || args.spaceId
     const environmentId = process.env.ENVIRONMENT_ID || args.environmentId
+    const { limit = 3, skip = 0 } = args
 
     const params = {
       spaceId,
       environmentId,
+      limit,
+      skip,
```

Comment:
These don't seem to work as-is, they'll need to be in the `query`.
```suggestion
      query: { limit, skip }
```